client/homebrew/editor/editor.less

```diff
@@ -4,41 +4,47 @@
 	width    : 100%;
 
 	.codeEditor{
-		height : 100%;
+		height        : 100%;
+		counter-reset : page;
 		.pageLine{
-			background-color : fade(#333, 15%);

```

Comment:
So I looked it up. `fade` is part of Less. Essentially takes the color and applies an opacity of x% to it.